src/redux/actions/samples/loadSamples.js

```diff
@@ -5,10 +5,18 @@ import {
 } from '../../actionTypes/samples';
 
 const loadSamples = (
-  experimentId,
+  experimentId = false, projectUuid = false,
 ) => async (dispatch) => {
   try {
```

Comment:
similarly to my comments about `loadProjects` -- make a new action type for `loading` state and dispatch it here before calling the API. Make sure you write reducers to handle it. Look at how `loadCellSets` is done.